### DISCUSSION TITLE
TS0505B Zigbee RGB+CCT light not acting as router

### DISCUSSION THREAD

**Discussion by merloose** [Koenkk/zigbee2mqtt#13405] [Devices]
Hi all,

I have purchased recently a bunch of TS0505B Zigbee RGB+CCT light . They pair fine with zigbee2mqtt. However they do not seem to act as routeur even so they are recognise as this type. Anybody has experience the same ?

I was hoping to use them all around the garden. But I need some of them to act as routeur to extend range properly.

Thanks guys in advance for sharing your experience.

**Comment by heliosgg** [comment]
Hi, it's not z2m problem. I've already created the same issue https://github.com/Koenkk/zigbee2mqtt/issues/13152

This happens even with tuya gateway in smartlife. I tried to contact smartlife support, they told that i should contact manufacturer of bulb. So I contacted seller, but i got nothing useful from him. I really want this problem to be fixed. I think the most possible solution is to find manufacturer of ts0505b and ask to fix firmware, so we will be able to update through tuya smart.
This is almost only one good rgb zigbee bulb ob the market, but with this bug problem

**Reply by merloose** [reply]
Thanks for the Update. Yes this is also a killer for me as the whole plan was to use them to extend range within the garden. I have tried with ZB-CL01 tuya bulb and same issue. I'll keep investigating.

**Comment by heliosgg** [comment]
Please mention me if you know what to do

**Reply by merloose** [reply]
I will thanks again !

**Comment by heliosgg** [comment]
@Koenkk, do you know, how we can find a REAL manufacturer of ts0505b?

**Reply by Koenkk** [reply]
`ts0505b` is manufacturer by TuYa

**Reply by heliosgg** [reply]
the problem is that tuya is not manufacturer of it. I contacted TuYa support through SmartLife, they directed me to seller and said that they doesnt manufacture smart devices. They just provide a cloud platform for other manufacturers of smart devices

**Reply by Koenkk** [reply]
From what I understood they are the manufacturer, they provide everything from hardware, software on the device and cloud platform (e.g. https://solution.tuya.com/projects/CMa4sxo0xr5g6i). One thing they don't provide is support ofc 😄

**Comment by eyk107** [comment]
Any news on this topic? Got the same issue
